run-tests.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+generate_id() {
+  LC_ALL=C tr -dc 'a-z' </dev/urandom | head -c 13 ; echo
+}
+
+port=9000
+containers=()
+sonarqube_container_name="it-sonarqube"
+network="it-network"
+
+print_usage() {
+  cat <<EOF
+usage: $0 [IMAGE...]
+
+examples:
+       $0 "sonar-scanner-cli:4.1"
+EOF
+}
+
+info() {
+  echo "[info] $@"
+}
+
+warn() {
+  echo "[warn] $@" >&2
+}
+
+fatal() {
+  echo "[error] $@" >&2
+  exit 1
+}
+
+require() {
+  local prog missing=()
+  for prog; do
+    if ! type "$prog" &>/dev/null; then
+      missing+=("$prog")
+    fi
+  done
+
+  [[ ${#missing[@]} == 0 ]] || fatal "could not find required programs on the path: ${missing[@]}"
```

Comment:
`${missing[*]}` is recommended within a string (see earlier comment)

Reply:
ok